platform/centec-arm64/docker-syncd-centec.mk

```diff
@@ -10,6 +10,9 @@ $(DOCKER_SYNCD_BASE)_DBG_DEPENDS += $(SYNCD_DBG) \
                                 $(LIBSAIMETADATA_DBG) \
                                 $(LIBSAIREDIS_DBG)
 
+SONIC_STRETCH_DOCKERS += $(DOCKER_SYNCD_BASE)
```

Comment:
still in stretch? when to move into buster?

Reply:
Thanks for the review comment.  I'll re-verify it on the buster. If it's ok, I'll move it into buster.

Reply:
it is fine, we can do it in a different pr.